File: Screamer.Application/Dtos/CreateNotificationDto.cs
namespace Screamer.Application.Dtos
{
    public class CreateNotificationDto
    {        public string Message { get; set; }

        public string Type { get; set; }

        public int ChatRoomId { get; set; }
        public string NotificationRoomId { get; set; }

        public int PostId { get; set; }

        public string SenderId { get; set; }
        public string RecieverId { get; set; }

        public int CommentId { get; set; }

        public int ReplyId { get; set; }

        public int ReactionId { get; set; }

        public int TagId { get; set; }

        public int MentionId { get; set; }

        public bool IsRead { get; set; }
    }
}

File: Screamer.AngularUI/src/app/core/models/CreateNotificationDto.ts
export interface CreateNotificationDto {
  message?: string;
  type?: string;
  chatRoomId?: number;
  notificationRoomId?: string;
  postId?: number;
  senderId?: string;
  recieverId?: string;
  commentId?: number;
  replyId?: number;
  reactionId?: number;
  tagId?: number;
  mentionId?: number;
  isRead?: boolean ;
}
